presto-docs/src/main/sphinx/develop/worker-protocol.rst

```diff
@@ -73,6 +77,9 @@ response includes:
   request the next chunk as ``X-Presto-Page-End-Sequence-Id`` HTTP header,
 * An indication that there are no more results as ``X-Presto-Buffer-Complete``
   HTTP header with the value of "true".
+* The remaining buffered bytes in the output buffer as ``X-Presto-Buffer-Remaining-Bytes``
+  HTTP header.  This may be used as a hint for upstream tasks to optimize data
```

Comment:
```suggestion
  HTTP header.  This can be used as a hint of upstream tasks to optimize data
```
Consider revising and maybe expanding this sentence to explain your meaning better. 
Maybe something like this: 
"This can be used as a sign of upstream tasks that would benefit from optimizing of their data exchange."